evalml/pipelines/components/transformers/samplers/undersampler.py

```diff
@@ -42,12 +41,32 @@ def __init__(
         random_seed=0,
         **kwargs,
     ):
+        if sampling_ratio <= 0 or sampling_ratio > 1:
```

Comment:
Is it worth getting ahead of the game and adding "raises" for these newish exceptions to the docstring?

Reply:
Love it! Unfortunately, the linting packages aren't smart enough to tell that the class docstring should be connected to the __init__ docstring but going to add this 😁